includes/payments/class-edd-payment.php

```diff
@@ -1110,6 +1013,14 @@ public function save() {
 		$customer = new EDD_Customer( $this->customer_id );
 		$customer->recalculate_stats();
 
+		$order_items = edd_get_order_items( array(
```

Comment:
Download recalculations are moved here with the customer recalculations--the new download calculations have to happen after the order items have been saved. This might actually be covered now with the `edd_transition_order_item_total` hook being added in but I've left it for now.